modules/honzon/src/lib.rs

```diff
@@ -307,6 +313,33 @@ pub mod module {
 			)?;
 			Ok(())
 		}
+
+		/// Transfers debit between two CDPs
+		///
+		/// - `from_currency`: Currency id that debit is transfered from
+		/// - `to_currency`: Currency id that debit is transfered to
+		/// - `debit_transfer`: Debit transfered across two CDPs
+		#[pallet::weight(<T as Config>::WeightInfo::transfer_debit())]
+		#[transactional]
+		pub fn transfer_debit(
+			origin: OriginFor<T>,
+			from_currency: CurrencyId,
+			to_currency: CurrencyId,
+			debit_transfer: Balance,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+			let debit_amount: Amount = debit_transfer.try_into().map_err(|_| ArithmeticError::Overflow)?;
+			let negative_debit = debit_amount.checked_neg().ok_or(ArithmeticError::Overflow)?;
+			<cdp_engine::Pallet<T>>::adjust_position(&who, from_currency, Zero::zero(), negative_debit)?;
```

Comment:
I think this requires user have enough aUSD on hand to close the debit?
We really want is user can still do this without having any free aUSD. Should have a test for this.